vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1292,6 +1293,24 @@ public void failure(VimeoError error) {
     }
 
     /**
+     * A package private search suggestions method. See {@link Search#suggest(String, int, int, ModelCallback)} for
+     * public usage of the suggestion API.
+     *
+     * @param queryMap the query parameters
+     * @param callback the callback to be invokedn when the call finishes
+     * @return the {@link Call} object provided by Retrofit
+     */
+    @Nullable
```

Comment:
This should be `@NotNull` as there's no way for the call to be nullable since we dereference it immediately after getting it from the service class. If the service can return null, then we should add a null-check around the dereference.